projects/charger/inc/charger_controller.h

```diff
@@ -10,12 +10,14 @@
 #include "can_interval.h"
 #include "charger_can.h"
 #include "generic_can.h"
+#include "gpio.h"
 
 typedef struct ChargerSettings {
   uint16_t max_voltage;
   uint16_t max_current;
   GenericCan *can;
   GenericCan *can_uart;
+  GPIOAddress relay_control_pin;
```

Comment:
Should create a storage struct for this pin and copy the data over

Reply:
Done.